packages/core-typings/src/Apps.ts

```diff
@@ -27,6 +27,27 @@ export type Tiers = {
 	price: number;
 };
 
+export type License = {
+	license: string;
+	version: number;
+	expireDate: string;
+};
+
+export type SubscriptionInfo = {
+	typeOf: string;
+	status: string;
+	statusFromBilling: boolean;
+	isSeatBased: boolean;
+	seats: number;
+	maxSeats: number;
+	license: License;
+	startDate: string;
+	periodEnd: string;
+	endDate: string;
+	externallyManaged: boolean;
+	isSubscribedViaBundle: boolean;
+};
+
```

Comment:
what do you think about keep as prefix like AppLicense and AppSubscriptionInfo ?